src/configupdater/block.py

```diff
@@ -34,18 +35,33 @@ class NotAttachedError(Exception):
     """{block} is not attached to a container yet. Try to insert it first."""
 
     def __init__(self, block: Union[str, "Block"] = "The block"):
-        msg = cast(str, self.__class__.__doc__).format(block=_short_repr(block))
+        doc = dedent(cast(str, self.__class__.__doc__))
```

Comment:
Very recently I also discovery 2 very handy functions `inspect.getdoc` and `inspect.cleandoc`!

(I am even using `cleandoc` in most of the places I would use `textwrap.dedent`, because it avoids the drama with with first line).

Reply:
Cool, haven't heard of them yet, I will replace them then.